api/v1alpha2/linodecluster_types.go

```diff
@@ -104,9 +104,24 @@ func (lm *LinodeCluster) SetConditions(conditions clusterv1.Conditions) {
 // NetworkSpec encapsulates Linode networking resources.
 type NetworkSpec struct {
 	// LoadBalancerType is the type of load balancer to use, defaults to NodeBalancer if not otherwise set
-	// +kubebuilder:validation:Enum=NodeBalancer
+	// +kubebuilder:validation:Enum=NodeBalancer;dns
```

Comment:
do we need another field for the eventual addition of using akamai or linode for DNS?

Reply:
done